src/dmd/dmodule.d

```diff
@@ -1473,11 +1472,8 @@ extern (C++) final class Module : Package
 
     extern (D) static void clearCache()
     {
-        for (size_t i = 0; i < amodules.dim; i++)
-        {
-            Module m = amodules[i];
+        foreach (Module m; amodules)
```

Comment:
`ref`?

Reply:
I tried to keep the code equivalent, even though it doesn't matter since `Module` is a class. I can change them all to `ref` if that's preferred.